### ISSUE TITLE
Comfort zones implementation

### ISSUE THREAD

**Issue by chemlabros** [psychrometrics/psychrolib#79]
**Is your feature request related to a problem? Please describe.**
Implement comfort zones based on Ashrae 55-2017 or EN 15251-2007

**Describe the solution you'd like**
Range of comfort zone between dry bulb temperature, relative humidity.

**Describe alternatives you've considered**
True or false statement that given array of [dryBulb, relHumidity] is inside comfort zone or not. 

**Additional context**

**Comment by dmey** [comment]
I don't think we would be looking to add this in PsychroLib but maybe @didierthevenard has a different view on this?

**Comment by chemlabros** [comment]
Maybe a function to get pmv/ppd (ashrae standar 55) given the humidity and temperature and some default values as a constant. 
This is a great deal to find optimal conditions.

**Comment by 6nv** [comment]
@chemlabros this is a very late reply, but check out [this psychrometric chart plugin](https://grafana.com/grafana/plugins/ventura-psychrometric-panel/) for the Grafana graphing platform. I created this using Psychrolib and built in ASHRAE comfort zones (for data centers, but the same philosophy can apply to human space comfort, if that's what you need.)
